packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+/* eslint-disable import/prefer-default-export */
+import LoggerProxy from '../common/logs/logger-proxy';
+
+import {getMaxFs, RemoteMedia, RemoteVideoResolution} from './remoteMedia';
+import {MediaRequestId, MediaRequestManager} from './mediaRequestManager';
+import {CSI, ReceiveSlot} from './receiveSlot';
+
+type Options = {
+  resolution?: RemoteVideoResolution; // applies only to groups of type MC.MediaType.VideoMain and MC.MediaType.VideoSlides
+  preferLiveVideo?: boolean; // applies only to groups of type MC.MediaType.VideoMain and MC.MediaType.VideoSlides
+};
+
+export class RemoteMediaGroup {
+  private mediaRequestManager: MediaRequestManager;
+
+  private priority: number;
+
+  private options: Options;
+
+  private remoteMedia: RemoteMedia[];
```

Comment:
I think this only contains unpinned remote media, it would be more readable if it was named accordingly

Reply:
I've renamed it and also removed `getPinnedRemoteMedia()` and changed `getRemoteMedia()` to take an argument to choose if client wants all, only pinned or only unpinned remote media

Reply:
that is nicer, would it also be nicer to have the same interface to includes, so you aren't checking pinned includes and unpinned includes in completely different ways?

Reply:
yeah, I agree, done.